[DVCSMP-2959] MSA-2127: Fibaro Door/ Window Sensor v2 **Official Request by Fibaro"

SmartThings opened the pull request:
Submitting 'Fibaro Door/ Window Sensor v2 **Official Request by Fibaro"' for publication

Commit:
MSA-2127: Fibaro Door/ Window 2 is a new contact sensor that was released in Europe in June 2017 and will be released in the USA on August 1st.  

D/W 2 has integrated temperature sensor and we revised how parameters are adjusted.

jackchi commented:
@parijatdas  Do you guys have this device in SRIB?

parijatdas commented:
@jackchi We don't have this device. We only have the old Fibaro Door/Window Sensor.

dkirker commented:
We have two in Mountain View.

dkirker commented:
@jackchi Before this gets merged (or just following it) we will want to tweak a few things so they are consistent with the other 4 Fibaro ZW5 device types.

dkirker commented:
I am looking this over today, I have a few changes that I will be adding.

dkirker committed:
Updated some event values for consistency with other Fibaro DTH
Changed some code style to also match other Fibaro DTH
Added initial state events

mckeed commented on devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy:
```diff
@@ -0,0 +1,516 @@
+		capability "Sensor"
+		capability "Health Check"
+	
+		attribute "temperatureAlarm", "string"
```
Looks like `attribute "multiStatus"` should be here too

mckeed commented on devicetypes/fibargroup/fibaro-door-window-sensor-2.src/fibaro-door-window-sensor-2.groovy:
```diff
@@ -0,0 +1,516 @@
+						break
+					case 6:
+						map.value = "underheat"
+						map.descriptionText = "Tamper alert: sensor removed or covering opened"
```
All the descriptionText values for temperature alarms are wrong

dkirker replied:
Ha! Good catch! Good ol' copy and paste...

dkirker commented:
@mckeed Fixed! Thanks for catching the copy/paste mess!

mckeed reviewed (approved):
I haven't really reviewed the configuration sync system, but it can't really be worse than the way we usually do it.

dkirker commented:
I jumped the gun. This isn't quite certified (I talked with the team last week and it should be there this week, I think).
